training/train.py

```diff
@@ -232,6 +232,16 @@ def train(
     logging.info("Saving model and optimizer state at iteration {} to {}".format(iteration, checkpoint_path))
 
 
+def calc_avgmax_attention(batch, y_pred):
```

Comment:
Please add a docstring

Reply:
little busy, do it for me if you can, overwise wait for me

Reply:
I'm not sure I entirely understand the process so I'll leave this to you if that's OK